src/Engine/Shaders/ParticleShader.cpp

```diff
@@ -0,0 +1,273 @@
+#include "ParticleShader.hpp"
```

Comment:
Yeah, cause Austin wants me to use Diffuse Frag.

Reply:
@Daskie any ideas for cutting down on duplicated code? Or screw it, we have deadlines coming up?

Reply:
shit, i forgot about this. The idea of using diffuse frag was so that you could just use particles_vert.glsl and diffuse_frag.glsl for the shader source. All this duplicate code I think comes down to engine limitations revolving around shaders that at this point will probably not be solved. Jaafer's right, deadlines are real, i don't care anymore as long as it looks good. If you want to rip all this out and just have some custom frag for particles, be my guest